src/LINEBot/MessageBuilder/TemplateBuilder/ButtonTemplateBuilder.php

```diff
@@ -113,8 +113,8 @@ public function buildTemplate()
             'actions' => $actions,
         ];
 
-        if ($this->title) {
-            $this->template['title'] = $this->title;
+        if (((string)($this->title)) !== "") {
+            $this->template['title'] = (string)($this->title);
```

Comment:
Please use `isset` instead.
```suggestion
        if (isset($this->title)) {
            $this->template['title'] = $this->title;
```
https://www.php.net/manual/en/types.comparisons.php

Reply:
Thank you for the suggestion.
However, when `$this->title = ""`, title included the template.
Message including empty text is rejected from MessagingAPI.
```
{
    "message": "A message (messages[0]) in the request body is invalid",
    "details": [
        {
            "message": "must be non-empty text",
            "property": "template/columns/0/title"
        }
    ]
}
```
It looks good to exclude the case `$this->title = ""`.

Reply:
Basically empty strings should be treated as set.
And we don't implement server side validation on the SDK include length of the strings, data structure and so on.
But we should implement input data handling consistently though. (Some class use `empty`)